portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,15 +14,15 @@
 
 package com.google.sps.servlets;
 
+import com.google.appengine.api.datastore.FetchOptions;
```

Comment:
Nit: alphabetize the import

Reply:
Oops I accidentally left that. Fixed it by putting it below Entity import line